src/resources/Props.ts

```diff
@@ -20,6 +20,7 @@ export interface ClerkProps {
 
 export interface ClientProps extends ClerkProps {
   sessionIds: string[];
+  // sessions: SessionProps[];
```

Comment:
❓

Reply:
I had a chat with @yourtallness , it's the convention he's using in the Node SDK in general. So, I thought it's better to keep it consistent.
For example: 
* https://github.com/clerkinc/clerk-sdk-node/blob/main/src/resources/Props.ts#L41-L42
* https://github.com/clerkinc/clerk-sdk-node/blob/main/src/resources/Props.ts#L123-L125
* ....

Basically, these attributes should be in the `Props`. But because of the way the types are structured, this is not possible, they are added one level below...these comments is a way to communicate that.

Reply:
Heh, this is confusing but I told @alex-ntousias to do it.

Just to convey there's a relationship association.

Actual association to concrete Sessions is declared in Client class.

We should revisit the types at some point, because I'm not confident the current modelling is optimal.